templates/repo/release/list.tmpl

```diff
@@ -75,6 +75,7 @@
 				</li>
 			{{end}}
 		</ul>
+		{{template "admin/base/page" .}}
```

Comment:
A agree that it is in the wrong place, but it does the right thing :smile: 

IMO it should be **moved** to `base/pagination.tmpl` and _everything_ should use that. (Pagination should not require customization...)